resize.js

```diff
@@ -103,7 +103,15 @@ function splitUrl(url) {
   return url.match(/^\/(.*)_(\d+)_(\d+)(_(\d+)x)?\.(.*)/);
 }
 
-function getImageParams(url) {
+function getImageParams(req) {
+  var url = req.url;
+  var queryParams = null;
+  var split = url.indexOf('?');
```

Comment:
Perhaps you need some additional module I don't know about, but I've printed `req` itself (and using various libraries) to the console. None of the times, did the query params show up.